src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
```

Comment:
Using `useParams` directly inside a component can lead to tight coupling with the routing logic. Consider passing `slug` as a prop to `PersonLink` for better reusability and testing.